man/machinery-man.1.md

```diff
@@ -0,0 +1,10 @@
+
+## man — Shows man page
+
+### SYNOPSIS
+
+  `machinery [global options] man` 
+
+### DESCRIPTION
+
+The `man` command shows the machinery man page.
```

Comment:
`Machinery` is written in other man pages with an upper-case `M` because Machinery is a name.